udata/core/dataset/api_fields.py

```diff
@@ -44,6 +44,9 @@
         required=True, enum=RESOURCE_TYPES.keys()),
     'format': fields.String(description='The resource format', required=True),
     'url': fields.String(description='The resource URL', required=True),
+    'permalink': fields.String(description='The resource permanent URL to '
+                               'the latest version (won\'t change on update)',
```

Comment:
We can be more explicit (I will try), but:
- I won't specify the 302 behavior which seems an internal to me
- On the second sentence, this is not exactly true: URL change only on data update. If someone update the resource description, data behind the URL won't change.

Proposal:

> The permanent URL to the latest version of the resource.
> When the resource data is updated, the URL will change, the permalink won't.

?